Makefile

```diff
@@ -105,3 +105,33 @@ clean:
 	cargo clean
 
 .PHONY: build build-debug run-local help clean all $(TEST_CHECK_PALLETS)
+
+# Start a 4 node CORD network using local chain spec
+.PHONY: spinup spindown
+
+NODE_1_CMD := ./target/production/cord --base-path /tmp/data/alice --validator --chain local --alice --port 30333 --rpc-port 9933 --prometheus-port 9615 --node-key abe47f4e1065d4aa6fb0c1dd69a9a6b63c4551da63aad5f688976f77bd21622f --rpc-methods=Safe --rpc-cors all --no-hardware-benchmarks --state-pruning 100 --blocks-pruning 100 --offchain-worker never --prometheus-external --telemetry-url "wss://telemetry.cord.network/submit/ 0"
+
+NODE_2_CMD := ./target/production/cord --base-path /tmp/data/bob --validator --chain local --bob --port 30334 --rpc-port 9934 --node-key 7609333b3e2e2e0c1b4064f074a7396b53d213e08d356d1be2d48fab3a6cd25a --rpc-methods=Safe --rpc-cors all --no-hardware-benchmarks --state-pruning 100 --blocks-pruning 100 --offchain-worker never --prometheus-external --telemetry-url "wss://telemetry.cord.network/submit/ 0" --bootnodes /ip4/127.0.0.1/tcp/30333/p2p/12D3KooWSNT7EqipGHpsAYptQfPNrMXdJcgjMd25hnQWwyvHxYnz
+
+NODE_3_CMD := ./target/production/cord --base-path /tmp/data/charlie --validator --chain local --charlie --port 30335 --rpc-port 9935 --node-key e18d2c105ad8188830979b7bf4e7779361beb9010b6574e1b35a0a354ce02e96 --rpc-methods=Safe --rpc-cors all --no-hardware-benchmarks --state-pruning 100 --blocks-pruning 100 --offchain-worker never --prometheus-external --telemetry-url "wss://telemetry.cord.network/submit/ 0" --bootnodes /ip4/127.0.0.1/tcp/30333/p2p/12D3KooWSNT7EqipGHpsAYptQfPNrMXdJcgjMd25hnQWwyvHxYnz
+
+NODE_4_CMD := ./target/production/cord --base-path /tmp/data/dave --chain local --dave --port 30336 --rpc-port 9936 --node-key f21d3114273b5d6184f9e595dba1850eb64b1e4965cfd2c6130354c67f632f5d --rpc-methods=Safe --rpc-cors all --no-hardware-benchmarks --state-pruning 100 --blocks-pruning 100 --offchain-worker never --prometheus-external --telemetry-url "wss://telemetry.cord.network/submit/ 0" --bootnodes /ip4/127.0.0.1/tcp/30333/p2p/12D3KooWSNT7EqipGHpsAYptQfPNrMXdJcgjMd25hnQWwyvHxYnz
+
+
+spinup: build
+	@echo "Starting nodes in the background..."
+	@$(NODE_1_CMD) &
+	@$(NODE_2_CMD) &
+	@$(NODE_3_CMD) &
+	@$(NODE_4_CMD) &
+	@echo "Four CORD nodes (Alice, Bob, Charlie, Dave) have been successfully started."
+	@echo "See them in \033[0;34mtelemetry.cord.network\033[0m under Cord Spin tab."
+
+spindown:
+	@echo "Stopping all nodes..."
+	@pkill -f "cord --base-path /tmp/data/"
+	@echo "All nodes stopped."
+	@echo "Deleting /tmp/data/ directory..."
+	@rm -rf /tmp/data/
+	@echo "/tmp/data/ directory deleted."
+	@echo "Bye Bye"
```

Comment:
> I feel 'Bye Bye' (twice) is not a great ending message, instead lets do something like "You can also connect with Dhiway to setup a private chain ([sales@dhiway.com](https://mail.google.com/mail/?view=cm&fs=1&tf=1&to=sales@dhiway.com))"
> 

If this is the code licensed under open source and available then co-mingling a commercial message might not be well received. If it is not, then something like "Commercial Support Services on CORD are offered by Dhiway <email>" might work

> "CORD team recommends having a separate chain in production, because 'local' chain uses the default keys, which are common."
> 

This is a very important message to convey

> Something like above...
> 
> @smohan-dw @NiranjanAP @sankarshanmukhopadhyay